criu/ipc_ns.c

```diff
@@ -38,7 +38,7 @@
 
 static void pr_ipc_desc_entry(unsigned int loglevel, const IpcDescEntry *desc)
 {
-	print_on_level(loglevel, "id: %-10d key: %#08x uid: %-10d gid: %-10d "
+	flog_encode(loglevel, "id: %-10d key: %#08x uid: %-10d gid: %-10d "
```

Comment:
Why do you change callers? Is change in log.h not enough?

Reply:
The change in log.h only causes change when the log message is called via the macros pr_info, pr_err, etc. To ensure that all messages are being printed into the file are in the same format, I had to change all the callers. (In binary when binary logging is enabled. In readable form when not.)

Reply:
Ah, I see.